src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -249,6 +278,13 @@ std::chrono::microseconds SQLPipelineStatement::execution_time_microseconds() co
   return _execution_time_micros;
 }
 
+std::string SQLPipelineStatement::get_time_string() const {
```

Comment:
When I saw this in `SQLPipeline`, I was tempted to ignore it, but now that we have the same code again here...
I don't think that this should be the responsibility of the SQLPipeline(Statement). Moving it to the caller would keep us from that duplication.

Reply:
if we move the responsibility to the caller, we have the duplication in the caller code. in theory the method can be removed from the SQLPipelineStatement, because that is never really used explicitly and is always wrapped by the SQLPipeline. Would that be ok?